src/components/GameView/GameUnavailableOverlay.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <v-overlay
+    data-cy="unavailable-game-overlay"
+    id="unavailable-game-scrim"
+    v-model="unavailableGame"
+    class="game-overlay text-center d-flex justify-center align-center"
+  >
+    <div class="overlay-header">
+      <h1 :class="[this.$vuetify.display.xs === true ? 'text-h5' : 'text-h3', ['font-weight-bold']]">
+        Oops!
+      </h1>
+      <p>Looks like this game isn't available</p>
+    </div>
+    <v-btn
+      color="secondary"
+      class="mt-4"
+      data-cy="leave-unavailable-game-button"
+      :loading="leavingGame"
+      @click="this.$router.push('/')"
+    >
+      Go Home
+    </v-btn>
+  </v-overlay>
+</template>
+
+<script>
+export default {
+  name: 'GameUnavailable',
+  props: {
+    unavailableGame: {
+      type: Boolean,
+      required: true,
+    },
+  },
+};
+</script>
+
+<style scoped lang="scss">
+:deep(.v-overlay__content) {
+  left: 0;
+}
+.game-overlay {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+}
+.overlay-header {
+  font-weight: bold;
+  background-color: rgba(var(--v-theme-surface-2));
+  color: rgba(var(--v-theme-surface-1));
+  padding: 24px;
+  text-align: center;
+  width: 100vw;
+}
```

Comment:
Now that we're using these styles in another place, it'd be good to define a `BaseOverlay` component that can accommodate this and the component here as well as the styles in the existing `GameOverlays`. That can reasonably be scoped into a separate issue, though

Reply:
https://github.com/cuttle-cards/cuttle/issues/430